custom_components/tuya_local/devices/solareast_aurora2_heatpump.yaml

```diff
@@ -38,7 +38,7 @@ entities:
           - dps_val: heating
             value: heat
           - dps_val: floor_heating
-            value: floor_heat
+            value: heat_cool
```

Comment:
I don't think you should change this here. There is no restriction on what the values in a select should be, so they should be correct, even if it does not directly match the climate hvac_mode.